File: AP/09_05_2024_Python_Estruturas_de_Dados_Listas/ex1.py
num = int(input("digite o numero"))
lista = []
while num != 0:
    lista.append(num)
    num = int(input("digite o numero"))
menor = lista[0]
maior = lista[0]
for i in range(1, len(lista)):
    if menor > lista[i]:
        menor = lista[i]
    if maior < lista[i]:
        maior = lista[i]
print(lista)
print(f"maior numero: {maior}")
print(f"menor numero: {menor}")

File: 1 Termo/ATP/1_termo_2_bimestre_ATP/Lista_9-Matrizes/ex1.c
 #include <stdio.h>
 int main(){
    int ma[3][3], maior=0;
    for(int i=0;i<3;i++){
        for(int j=0;j<3;j++){
            scanf("%d", &ma[i][j]);
        }
    }
    for(int i=0;i<3;i++){
        for(int j=0;j<3;j++){
            maior = (maior < ma[i][j]) ? ma[i][j] : maior;
        }
    }
    printf("maior numero eh %d\n", maior);
    return 0;
 }